File: backend/src/Modules/Modules.Invitation.Application/Invitations/GetAdminList/AdminInvitationListItem.cs
namespace Modules.Invitation.Application.Invitations.GetAdminList;

public sealed record AdminInvitationListItem(
    Guid Id,
    string TitleHu,
    string? TitleEn,
    int? Year,
    bool HasImage,
    bool IsPublished,
    int DisplayOrder,
    DateTime CreatedAtUtc);

File: frontend/henn-laszlo/projects/invitation-data-access/src/lib/models/admin-invitation-list-item.ts
export interface AdminInvitationListItem {
  readonly id: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly hasImage: boolean;
  readonly isPublished: boolean;
  readonly displayOrder: number;
  readonly createdAtUtc: string;
}